src/profile_manager/templates/profile_manager/profile_list.html

```diff
@@ -167,7 +171,9 @@
 
                 {% if view_type != VIEW_TYPES.STAFF and view_type != VIEW_TYPES.INACTIVE %}
 
-                    <td>{{ object.grad_year|default:"-" }}</td>
+                    {% if config.custom_profile_field %}
```

Comment:
Same as earlier comment, does this trigger with spaces?  If it's not a problem, good.  If it is a problem, maybe create a separate method that checks if this field exists that can be used where needed.